test/shared/models/utils/JsonErrorValidators.scala

```diff
@@ -94,6 +118,21 @@ trait JsonErrorValidators {
     }
   }
 
+  def testOptionalFields[A](json: JsValue)(property: String)(implicit rds: Reads[A]): Unit = {
+    s"the JSON is missing the optional property $property" should {
+
+      val jsonWithoutProperty = json.as[JsObject].-(property)
+
+      "not throw any errors" in {
+        jsonWithoutProperty.validate[A].isError shouldBe false
```

Comment:
Can these be assigned to a "val result", and then "result shouldBe ...", more in line with the test pattern we generally use